prow/monitoring/grafana_expose.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: grafana
+  name: grafana
+  namespace: prow-monitoring
+spec:
+  ports:
+  - name: http
+    port: 3001
+    targetPort: http
+  selector:
+    app: grafana
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: monitoring-prow
+    kubernetes.io/tls-acme: "true"
+    certmanager.k8s.io/cluster-issuer: letsencrypt-prod
+    certmanager.k8s.io/acme-http01-edit-in-place: "true"
+    kubernetes.io/ingress.class: "gce"
+  name: grafana
+  namespace: prow-monitoring
+spec:
+  tls:
+    - secretName: grafana-tls
```

Comment:
Will this secret be automatically created?

Reply:
Looks like yes

Reply:
https://github.com/kubernetes/test-infra/pull/13003#discussion_r293126258